redis/_cache.py

```diff
@@ -160,7 +160,27 @@ class EvictionPolicy(Enum):
     RANDOM = "random"
 
 
-class _LocalCache:
+class AbstractCache:
```

Comment:
WDYT: Have a look at Abstract base classes, and maybe inherit from [ABC](https://docs.python.org/3/library/abc.html#abc.ABC)?